ext/compressors/zstd/zstd_compress.c

```diff
@@ -192,15 +296,104 @@ zstd_pre_size(
     return (0);
 }
 
+/*
+ *  zstd_init_context_pool --
+ *	Initialize a given type of context pool.
+ */
+static int
+zstd_init_context_pool(
+  ZSTD_COMPRESSOR *zcompressor, CONTEXT_TYPE ctx_type, int count, ZSTD_CONTEXT_POOL **ctx_pool)
```

Comment:
I think `ctx_pool` could be named `context_poolp` to make the naming a bit easier to follow. Similarly below in the terminate function.